File: BrahMos/AlgoMap/Arrays & Strings/1768_Merge_Strings_Alternately.cpp
class Solution
{
public:
    string mergeAlternately(string word1, string word2)
    {
        // vector<int>ans;
        string ans = "";
        // for(int i=0;i<n;i++){
        int i = 0, j = 0;
        int n = word1.size();
        int m = word2.size();
        while (i < n or j < m)
        {
            if (i < n)
            {
                ans.push_back(word1[i]);
                i++;
            }
            if (j < m)
            {
                ans.push_back(word2[j]);
                j++;
            }
        }
        return ans;
    }
};

File: BrahMos/AlgoMap/Arrays & Strings/1768_Merge_Strings_Alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        n=len(word1)
        m=len(word2)
        i=j=0
        ans=[]
        while(i<n or j<m):
            if(i<n):
                ans.append(word1[i])
                i+=1
            if(j<m):
                ans.append(word2[j])
                j+=1
        return "".join(ans)